tests/component/conftest.py

```diff
@@ -156,7 +156,7 @@ def di_single_channel_multi_line_task(
 ) -> nidaqmx.Task:
     """Configure a single-channel multi-line digital input task."""
     task.di_channels.add_di_chan(
-        flatten_channel_string(sim_6363_device.di_lines.channel_names[:8]),
+        flatten_channel_string(sim_6363_device.di_lines.channel_names[:8][::-1]),
```

Comment:
> In conftest.py, all the channel strings used with CHAN_FOR_ALL_LINES in add_di_chan() have been reversed (using [::-1]), so they count down instead of up. (Note that this matches the behavior of port expansions)

I don't think you should do this. Users typically do not specify reversed channel strings. Also, port expansions are only reversed for waveforms, not for read_digital_lines.